packaging/datadog-agent/source/install_agent.sh

```diff
@@ -144,7 +144,11 @@ else
 fi
 
 printf "\033[34m* Starting the Agent...\n\033[0m\n"
-$sudo_cmd /etc/init.d/datadog-agent restart
+if [ $OS = "Debian" ]; then
+    $sudo_cmd invoke-rc.d datadog-agent restart
+else
+    $sudo_cmd /etc/init.d/datadog-agent restart
+fi
```

Comment:
Seems like a good idea. Ultimately it would be good to remove the system related if blocks, apart from the package manager related ones in the source install script ofc.